src/Backend/Modules/Extensions/Installer/Installer.php

```diff
@@ -126,39 +80,24 @@ public function install(): void
                 'extensions/edit_theme_template',
             ]
         );
-
-        if ($this->installExample()) {
-            $this->installForkTheme();
-        }
     }
 
-    private function setRights(): void
+    private function configureBackendRights(): void
     {
-        // modules
-        $this->setActionRights(1, $this->getModule(), 'Modules');
-        $this->setActionRights(1, $this->getModule(), 'DetailModule');
-        $this->setActionRights(1, $this->getModule(), 'InstallModule');
-        $this->setActionRights(1, $this->getModule(), 'UploadModule');
-
-        // themes
-        $this->setActionRights(1, $this->getModule(), 'Themes');
-        $this->setActionRights(1, $this->getModule(), 'DetailTheme');
-        $this->setActionRights(1, $this->getModule(), 'InstallTheme');
-        $this->setActionRights(1, $this->getModule(), 'UploadTheme');
+        $this->setModuleRights(1, $this->getModule());
 
-        // templates
-        $this->setActionRights(1, $this->getModule(), 'ThemeTemplates');
-        $this->setActionRights(1, $this->getModule(), 'AddThemeTemplate');
-        $this->setActionRights(1, $this->getModule(), 'EditThemeTemplate');
-        $this->setActionRights(1, $this->getModule(), 'DeleteThemeTemplate');
+        $this->configureBackendActionRightsForModules();
+        $this->configureBackendActionRightsForTemplates();
+        $this->configureBackendActionRightsForThemes();
     }
 
-    private function installForkTheme(): void
+    private function configureFrontendExtras(): void
     {
-        /*
-         * Fallback templates
-         */
+        $this->insertExtra('Search', ModuleExtraType::widget(), 'SearchForm', 'Form');
```

Comment:
Because of this the widget is added more than once in the database, is this necessary here?